mikupad.html

```diff
@@ -7130,6 +7229,26 @@
 					Export prompt to plaintext
 				</button>
 			</div>
+			<${Checkbox} label="Enable Text-to-Speech"
+						disabled=${!!cancel} value=${window.TTS.enabled} onValueChange=${changeTTSEnabled}/>
+			<div className="hbox-flex" style=${{ "flex-wrap": "unset" }}>
+					<${SelectBox}
+						id="voices"
+						label="Voice"
+						disabled=${!!cancel}
+						value=${window.TTS.voice_id}
+						onValueChange=${selectNewVoice}
+						options=${listTTSVoices}/>
+					<button title="Stop TTS" className="symbol-button" disabled=${cancel} onClick=${ttsStop}>
+						<${SVG_Cancel} style=${{ 'width': '.95em', 'transform': 'translate(-50%, -45%)' }}/>
+					</button>
+				</div>
+			<${InputSlider} label="TTS Pitch" type="number" step="0.1" max="2"
+				readOnly=${!!cancel} value=${window.TTS.pitch} onValueChange=${changeTTSPitch}/>
+				<${InputSlider} label="TTS Rate" type="number" step="0.1" max="2"
+				readOnly=${!!cancel} value=${window.TTS.rate} onValueChange=${changeTTSRate}/>
+				<${InputSlider} label="TTS Volume" type="number" step="0.1" max="2"

```

Comment:
This doesn't look good in the interface... I think a good improvement would be to hide the TTS-related elements when TTS is disabled.